plugin/src/main/java/me/block2block/hubparkour/listeners/FallListener.java

```diff
@@ -47,11 +47,10 @@ public void onFall(EntityDamageEvent e) {
                             return;
                         }
                     }
-                } else {
-                    if (ConfigUtil.getBoolean("Settings.Health.Disable-Damage", true)) {
-                        e.setCancelled(true);
-                    }
-                    return;
+                }
+
+                if (ConfigUtil.getBoolean("Settings.Health.Disable-Damage", true)) {
```

Comment:
Moving this has other consequences and may not work as intended. With the current implementation, this will only execute:
 - when due to void damage, teleport on void is enabled and the player hasn't been teleported by another plugin.
 - when due to fall damage and teleport on fall is enabled, or the player hasn't fallen far enough

Any other circumstances, this will not execute (so basically any other damage)

Reply:
`Settings.Health.Disable-Damage` is not `Disable All Damage`?

Reply:
My understanding is to turn off all the damage caused

Reply:
You would be correct, but due to the logic of the if statement you took it out of, it will no longer cover other scenarios other than the ones I've listed.